tests/budget_accounting_test.py

```diff
@@ -529,6 +522,51 @@ class ComputeBudgetTestCase:
                         msg=f"failed test {case.name} expected mechanism delta "
                         f"{expected_mechanism_delta} "
                         f"got {actual_mechanism._delta}")
+                else:  # mechanism != Generic
+                    self.assertAlmostEqual(
+                        first=expected_mechanism_noise_std,
+                        second=actual_mechanism.noise_standard_deviation,
+                        delta=1e-2,
+                        msg=f"failed test {case.name} expected mechanism noise "
+                        f"{expected_mechanism_noise_std} "
+                        f"got {actual_mechanism.noise_standard_deviation}")
+
+    def test_compute_budgets_gaussian_thresholding(self):
+        accountant = PLDBudgetAccountant(total_epsilon=1.0, total_delta=1e-8)
+        thresholding_budget = accountant.request_budget(
+            MechanismType.GAUSSIAN_THRESHOLDING)
+        guasisan_budget = accountant.request_budget(MechanismType.GAUSSIAN)
+        accountant.compute_budgets()
+
+        # Gaussian mechanism thresholding and Gaussian mechanism have to have the
+        # same sigma, since Gaussian mechanism thresholding is modeled with
+        # Gaussian.
+        self.assertAlmostEqual(thresholding_budget.noise_standard_deviation,
```

Comment:
why not to assert that `thresholding_budget.noise_standard_deviation == guasisan_budget.noise_standard_deviation`?